client/src/search/Typeahead/Typeahead.js

```diff
@@ -194,55 +194,56 @@ export class Typeahead extends React.Component {
                     </a>
                   </li>
                 )}
-                {_.chain(results_on_page)
-                  .groupBy("config_group_index")
-                  .flatMap((results, group_index) =>
-                    _.map(results, (result, index) => ({
-                      is_first_in_group: index === 0,
-                      group_index,
-                      result,
-                    }))
-                  )
-                  .flatMap(
-                    (
-                      { is_first_in_group, group_index, result },
-                      result_index
-                    ) => {
-                      const adjusted_result_index = needs_pagination_up_control
-                        ? result_index + 1
-                        : result_index;
-
-                      return [
-                        is_first_in_group && (
+                {!_.isEmpty(this.config_groups) &&
```

Comment:
This check isn't actually needed, but I left it there just in case something goes wrong